routes/mentors.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+
+const router = express.Router();
+const mentorService = require("../services/mentor_service");
+const { ROLE_NAMES } = require("../utils");
+const {verifyRole} = require("../middleware/authorization");
+const { isValidIntegerId, respondWithError } = require("./helpers");
+const logger = require('../logger')(__filename);
+
+router.use(express.json());
+ 
+// Get all mentors
+router.get("/mentors", async (req, res) => {
+  try {
+    const { status } = req.query;
+    const mentors = await mentorService.getMentors(req.userId, status);
+    res.json({ mentors });
+  } catch (err) {
+    if (err.message.includes('not permitted')) {
+      logger.warn(`Unauthorized mentor list request: ${err.message}`);
+      return respondWithError(res, 403, "You don't have permission to view these mentors");
+    }
+    logger.error(err);
+    respondWithError(res, 500, "Failed to retrieve mentors");
```

Comment:
In case this is a default 500, i think we can go with default "Internal Server Error" message, i.e. use
`respondWithError(res)`
the fact that we failed to retrieve mentors is pretty obvious here :D